torchvision/ops/poolers.py

```diff
@@ -2,10 +2,10 @@
 from torch import nn, Tensor
 
 import torchvision
-from torchvision.ops import roi_align
+from .roi_align import roi_align
```

Comment:
I would like to know why.

Reply:
mypy was confused with the import (since there is a `roi_align.py` file)
Since we're in the correct folder, switching to relative import cleared that!